Assets/Unity Assets/VillagePack/BigHouse/BigHouse.fbm.meta~HEAD

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
+guid: 5fdd8a753b8d545958d3500a14cc21ae
+folderAsset: yes
+DefaultImporter:
+  externalObjects: {}
+  userData: 
+  assetBundleName: 
+  assetBundleVariant: 
```

Comment:
Thoses files are garbage (*~HEAD, *~main), can you remove them ?

Reply:
I agree with this ^